kernelcraft/run.py

```diff
@@ -50,6 +50,9 @@ def make_parser():
 
     parser.add_argument('--config', '-f', action='append',
             help='Use one (or more) specific kernel .config snippet to override default config settings')
+    
+    parser.add_argument('--cpus', '-smp', action='store',
```

Comment:
second argument should be the short form of the option, typically just one letter, considering that -c and -s are used already maybe we can go with '-p' (that would stand for processors)?

Reply:
I went with `smp` exactly because the letters that made sense were all already taken, so I went with the QEMU argument name since it felt intuitive to use to me, switched to `-p`!